test/functional/apps/saved_objects_management/edit_saved_object.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+
+export default function({ getPageObjects, getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const PageObjects = getPageObjects(['common', 'settings']);
+
+  const setFieldValue = async (fieldName: string, value: string) => {
+    return testSubjects.setValue(`savedObjects-editField-${fieldName}`, value);
+  };
+
+  const getFieldValue = async (fieldName: string) => {
+    return testSubjects.getAttribute(`savedObjects-editField-${fieldName}`, 'value');
+  };
+
+  const focusAndClickButton = async (buttonSubject: string) => {
+    const button = await testSubjects.find(buttonSubject);
+    await button.scrollIntoViewIfNecessary();
+    await delay(10);
+    await button.focus();
+    await delay(10);
+    await button.click();
+  };
```

Comment:
I don't know the exact cause, but only `click`ing on the buttons was very flaky (it seems the texteditor are catching the focus / blocking the click during the driver  auto-scrolling to the button). I had to use that to remove the flakyness...